maze-info.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index abb1d02..1c4f40e 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -2,10 +2,17 @@
+ name = Maze
+ exec = sugar-activity3 activity.MazeActivity
+ icon = activity-maze
+-activity_version = 29
++activity_version = 2
++release_date = 2018-04-12
+ license = GPLv3+
+-bundle_id = vu.lux.olpc.Maze
+-summary = Good at finding your way out of hard situations? Let's see! Try to make your way through an increasingly difficult path, or you can also play with a friend!
+-max_participants = 6
++metadata_license = CC0-1.0
++update_contact = tch@sugarlabs.org
++bundle_id = org.sugarlabs.Maze
++summary = A simple maze game for the Sugar learning environment.
```

Comment:
the summary field should not end with that final dot. (only the description should include it, make sure to run the validators mentioned in the guide)